purchase_work_acceptance_invoice_plan/models/work_acceptance.py

```diff
@@ -0,0 +1,21 @@
+
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class WorkAcceptance(models.Model):
+    _inherit = "work.acceptance"
+
+    wa_in_id = fields.Many2one(
```

Comment:
```suggestion
    installment_id = fields.Many2one(
```
Let's change name of the wa_in_id, everywhere.